### ISSUE TITLE
incorrect order of executed operations

### ISSUE THREAD

**Issue by adasari** [ava-labs/gecko#262]
Dowloaded the 0.5.7 binary and run the node. During the X-chain bootstrap noticed the following log.

```
INFO [06-26|02:55:52] <X Chain> /snow/engine/avalanche/bootstrapper.go#330: executed 895000 operations
INFO [06-26|02:55:55] <X Chain> /snow/engine/avalanche/bootstrapper.go#330: executed 897500 operations
INFO [06-26|02:55:57] <X Chain> /snow/engine/avalanche/bootstrapper.go#330: executed 900000 operationsINFO [06-26|02:55:47] <X Chain> /snow/engine/avalanche/bootstrapper.go#330: executed 890000 operations
INFO [06-26|02:55:50] <X Chain> /snow/engine/avalanche/bootstrapper.go#330: executed 892500 operations
INFO [06-26|02:55:52] <X Chain> /snow/engine/avalanche/bootstrapper.go#330: executed 895000 operations
INFO [06-26|02:55:55] <X Chain> /snow/engine/avalanche/bootstrapper.go#330: executed 897500 operations
INFO [06-26|02:55:57] <X Chain> /snow/engine/avalanche/bootstrapper.go#330: executed 900000 operationsINFO [06-26|02:59:30] <X Chain> /snow/engine/avalanche/bootstrapper.go#330: executed 67500 operations
INFO [06-26|02:59:33] <X Chain> /snow/engine/avalanche/bootstrapper.go#330: executed 70000 operations
INFO [06-26|02:59:37] <X Chain> /snow/engine/avalanche/bootstrapper.go#330: executed 72500 operations
INFO [06-26|02:59:41] <X Chain> /snow/engine/avalanche/bootstrapper.go#330: executed 75000 operations
INFO [06-26|02:59:44] <X Chain> /snow/engine/avalanche/bootstrapper.go#330: executed 77500 operations
INFO [06-26|02:59:48] <X Chain> /snow/engine/avalanche/bootstrapper.go#330: executed 80000 operations

```

**Comment by danlaine** [comment]
Hi @adasari thanks for reporting.

I think the issue here is the way your terminal is presenting the logs, as opposed to an underlying issue in the code.
The reason I think this is because you have multiple logs on one line, and because some of the logs are chronologically out of order. This is the only report we've had of out-of-order logs, so I think it's more likely that the issue is on your machine.

As for the operation count going from 900000 to 7000: when bootstrapping a DAG, all transactions are executed, and then all vertices. As such, it's expected that the operation count resets after transactions are executed and you see something like this:
```
INFO [06-26|02:55:52] <X Chain> /snow/engine/avalanche/bootstrapper.go#330: executed 895000 operations
INFO [06-26|02:55:55] <X Chain> /snow/engine/avalanche/bootstrapper.go#330: executed 897500 operations
INFO [06-26|02:55:57] <X Chain> /snow/engine/avalanche/bootstrapper.go#330: executed 900000 operations
INFO [06-29|02:55:57] <X Chain> /snow/engine/avalanche/bootstrapper.go#333: executed 900123 operations
INFO [06-29|02:55:57] <X Chain> /snow/engine/avalanche/bootstrapper.go#298: executing vertex state transitions...
INFO [06-26|02:55:58] <X Chain> /snow/engine/avalanche/bootstrapper.go#330: executed 2500 operations
INFO [06-26|02:55:59] <X Chain> /snow/engine/avalanche/bootstrapper.go#330: executed 5000 operations
```  

What you're seeing is sorta kinda close to this, but again, I think your terminal may just be displaying things funny.

Have you been able to reproduce this at all?

**Comment by adasari** [comment]
i noticed this twice. let me try again if possible on different machine. thanks.

**Comment by danlaine** [comment]
Going to close this. I think this was something local to your machine. Let us know if you're able to reproduce this.
